apps/src/gamelab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -1,4 +1,4 @@
-var actions = require('./animationPickerModule');
+import * as actions from './animationPickerModule';
 var AnimationPickerBody = require('./AnimationPickerBody.jsx');
```

Comment:
Their reason,

> Why? This makes sure you have a single default export.

...sounds like a tautology?  In any case, I'm happy to conform here.  I'm tempted to leave the `*` import in AnimationPickerTest.js for now, simply because it eases interoperability.